test/lib/utils-spec.js

```diff
@@ -21,14 +21,43 @@ describe('utils', () => {
   });
   describe('Config', () => {
```

Comment:
you don't seem to have any tests about actually migrating a user's old creds file and what it should look like new